src/Automation/Interfaces/IScanner.cs

```diff
@@ -39,10 +60,28 @@ public interface IScanner
         /// All exceptions are wrapped in <see cref="AxeWindowsAutomationException"/>.
         /// If the exception was not thrown by AxeWindows automation, the <see cref="Exception.InnerException"/> property
         /// will contain the exception.
+        /// Calling this function is equivalent to passing false to <see cref="IScanner.Scan(string, bool)"/> and passing false for <c>enableMultipleWindows</c>.
         /// </remarks>
         /// <returns>Information about the scan and any issues detected</returns>
         /// <exception cref="AxeWindowsAutomationException"/>
         ScanResults Scan(string scanId);
```

Comment:
If someone calls this version with a `Config` that enables multiple roots, we should probably throw an error code telling them to call the other method. Otherwise we don't have a good way to return the data that they need.